src/CashuWallet.ts

```diff
@@ -388,12 +388,12 @@ class CashuWallet {
 	 * @param invoice LN invoice that needs to get a fee estimate
 	 * @returns the mint will create and return a melt quote for the invoice with an amount and fee reserve
 	 */
-	async meltQuote(invoice: string): Promise<MeltQuoteResponse> {
+	async createMeltQuote(invoice: string): Promise<MeltQuoteResponse> {
 		const meltQuotePayload: MeltQuotePayload = {
 			unit: this._unit,
 			request: invoice
 		};
-		const meltQuote = await this.mint.meltQuote(meltQuotePayload);
+		const meltQuote = await this.mint.postMeltQuote(meltQuotePayload);
```

Comment:
The idea was to have the mint's method reflect the actual protocol (e.g. making a POST request to the melt endpoint), while the wallet's method is more descriptive. However I do get the point that this might be confusing

Reply:
Done :)